### DISCUSSION TITLE
optimisation section

### DISCUSSION THREAD

**Discussion by Allygt** [PrestaShop/PrestaShop#37646] [Ideas]
### The problem

prestashop lacks a simple database maintenance tool, it could be nice from prestashop to be able to empty the tables for statistics, logs, and cache all in one screen

### The solution

the best thing is to create a native module for prestashop

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by Hlavtox** [comment]
While it would be nice, PrestaShop core already has a lot of functionality.

You can build a tool like this for your needs freely. I did for my clients, running integrity checks, repairs, cleanups every night.

**Reply by Allygt** [reply]
you have a store?

**Comment by kpodemski** [comment]
Hello @Allygt 

Of course, it would be great for the core to have it all, but if it is not possible, maintaining all of the features would be an enormous burden. That's why, since the software is open and we have a fantastic community, there are **a lot** of modules that can help you clean your database. There are even **free modules**, for example: https://github.com/SpiriitLabs/prestaclean
